### PR TITLE
Fix: Bad CRLF on some EWS server responses

### PR DESCRIPTION
**Pull request by nick4tech**
It seems in the wild there are some responses from EWS servers with incorrectly formatted mimes. Namely, they use `\r` instead of `\r\n` as line breaks. Sisimai doesn't properly parse these messages. This PR introduces a fix so that it can properly handle these bounces.

### CODE REVIEW

**Comment by azumakuniyuki** [comment]
@nick4tech Thanks for the pull request. Now, I am currently busy with "No Auth, No Entry" related tasks, so I will review the code next week.

**Comment by nick4tech** [comment]
@azumakuniyuki Hello! Please let me know when you've had a chance to review this 😄

File: lib/sisimai/message.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ class Message
     def initialize(data: '', **argvs)
       return nil if data.empty?
       param = {}
-      email = data.scrub('?').gsub("\r\n", "\n")
+      email = data.scrub('?').gsub("\r\n", "\n").gsub("\r", "\n")
       thing = { 'from' => '','header' => {}, 'rfc822' => '', ds => [], 'catch' => nil }
```
Comment (azumakuniyuki) [inline lib/sisimai/message.rb:38]: All the tests have passed successfully
```
% make test
rake spec
/usr/local/bin/ruby -I/usr/local/lib/ruby/gems/2.6.0/gems/rspec-support-3.9.0/lib:/usr/local/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/lib /usr/local/lib/ruby/gems/2.6.0/gems/rspec-core-3.9.0/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
...
Finished in 1 minute 24.86 seconds (files took 45.5 seconds to load)
241555 examples, 0 failures

make test  102.61s user 17.41s system 91% cpu 2:11.70 total
```

File: set-of-emails/invalid_bounce.txt [hunk 2]
```diff
@@ -0,0 +1 @@
...

----==_mimepart_65c1fd2aee07b_72f9497553
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit


----==_mimepart_65c1fd2aee07b_72f9497553--

----==_mimepart_65c1fd2aee126_72f949768--
```
Comment (azumakuniyuki) [inline set-of-emails/invalid_bounce.txt:1]: This file will be moved/renamed to `set-of-emails/maildir/<somewhere>` after merging code

**Comment by azumakuniyuki** [comment]
By the way, what is "EWS Servers"?

**Comment by nick4tech** [comment]
> By the way, what is "EWS Servers"?

It's a Microsoft product for mailing servers: https://learn.microsoft.com/en-us/exchange/exchange-server?view=exchserver-2019

**Comment by azumakuniyuki** [comment]
@nick4tech Thank you :-)

**Pull request merged**